cves/kernel/CVE-2013-5634.yml

```diff
@@ -223,7 +233,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: 'arch'
   note:
```

Comment:
Could be more specific, which directory within 'arch'?

Reply:
Thanks for the feedback, this vulnerability occurs within the arch/arm/kvm directory. I will add a note explaining it is within the ARM platform when KVM is used.